src/python/elements.cpp

```diff
@@ -1556,6 +1573,49 @@ void init_elements(py::module& m)
     ;
     register_beamoptics_push(py_TaperedPL);
 
+    py::class_<LinearMap, elements::Named, elements::Alignment, elements::LinearTransport> py_LinearMap(me, "LinearMap");
+    py_LinearMap
+        .def("__repr__",
+             [](LinearMap const & linearmap) {
+                 return element_name(
+                     linearmap,
+                     std::make_pair("R11", linearmap.m_transport_map(1,1)),
+                     std::make_pair("R12", linearmap.m_transport_map(1,2))
```

Comment:
The specification of R11 and R12 here is arbitrary.  These lines were added during testing before your fix to the Python bindings.  We should decide whether to modify or delete these.

Reply:
Good question, does it make sense to preview a few very common ones (R11, R12, R56)? Users can always print the matrix to see it all.

Reply:
Maybe it is confusing if we show selected entries, people might assume the other values are not set...